fl4health/strategies/basic_fedavg.py

```diff
@@ -248,12 +248,15 @@ def aggregate_fit(
         if not self.accept_failures and failures:
             return None, {}
 
-        # Convert results
-        weights_results = [
-            (parameters_to_ndarrays(fit_res.parameters), fit_res.num_examples) for _, fit_res in results
+        # Sorting the results by elements and sample counts. This is primarily to reduce numerical fluctuations in s
```

Comment:
I think in all of the comment `in s` left extra.

Reply:
Yup. You are correct. This will be everywhere this comment was duplicated. Will fix.